process/block/preprocess/transactions.go

```diff
@@ -344,7 +350,25 @@ func (txs *transactions) computeTxsToMe(body *block.Body) ([]*txcache.WrappedTra
 				miniBlock.ReceiverShardID)
 		}
 
-		txsFromMiniBlock, err := txs.computeTxsFromMiniBlock(miniBlock)
+		miniBlockHash, err := core.CalculateHash(txs.marshalizer, txs.hasher, miniBlock)
+		if err != nil {
+			return nil, err
+		}
+
+		indexOfLastTxProcessedByItself := int32(-1)
```

Comment:
It means the last index of the transaction processed by self in the given mini block received from the proposer in the proposed block. Then we have indexOfLastTxProcessedByProposer taken for the miniblock header included in the proposed block. Based on this two values the validator will try to process all the transactions with index > indexOfLastTxProcessedByItself and index <= indexOfLastTxProcessedByProposer and check if it will reach to the same results. If the result does not match the node will not sign the proposed block. Actually it will not trust the proposer, and will not take the indexOfFirstTxProcessed from the mini block header, but only the indexOfLastTxProcessed, just to avoid some malicious situations when the proposer will put there intentionally a greater index for the first tx processed, therefore skipping execution of some txs from the mini block. So the node will always use its "index" to continue the processing of a partial mini block.

Reply:
Any suggestion for renaming?